packages/vlossom/src/components/vs-render/__tests__/vs-render.test.ts

```diff
@@ -0,0 +1,189 @@
+import { describe, it, expect } from 'vitest';
+import { mount } from '@vue/test-utils';
+import VsRender from './../VsRender.vue';
+import { markRaw } from 'vue';
+
+describe('VsRender', () => {
```

Comment:
P2: 이해한 바가 맞다면 `VsRender`의 큰 장점이 html dom tag를 동적으로 적용할 수 있다는 점이라고 생각해서,       
**관련 테스트를 추가하는 것을 제안 드리고 싶습니다.**     

컴퍼넌트가 javascript의 eval을 떠오르게 해요
반대로, 위 경우가 아닌 경우 `<compoment />`, `v-condition`, `v-loop`로 대응가능하지 않은가 하는 감상도 들었습니다.

```suggestion

// suedo
describe('VsRender', () => {
	describe('문자열 content 렌더링', () => {
      	it('HTML 태그가 포함된 문자열을 컨텐츠로 사용하여, 동적으로 DOM을 랜더링 할 수 있다', () => {
        	//given
        	const buttonTag = ref('<button (click)=handleA />');
        	const computed = computed (() => buttonTag.value);
	        const wrapper = mount(VsRender, {        		        
           	props: {
           		   	content: buttonTag.value,
	            },
            });
	            
			// when
			// buttonTag에 이벤트 네임 갈아끼우기
					
  		    // then
            expect(wrapper).shouldBe(갈아끼워진 이벤트가 있는 버튼);  
		}
	})
}
```

Reply:
`'복잡한 HTML 구조가 주어지면 올바른 구조로 렌더링되어야 한다'`라는 테스트에서 어느 정도 html attribute들이 잘 들어가는 건 확인 할 수 있구요.<br />
아마 event 걸어두는 방식을 [onclick 같은 스태틱 html 방식](https://www.w3schools.com/jsref/event_onclick.asp)으로 생각하시는 것 같은데, 필요하다면 이 방식보다는 window.addEventListener 방식이 더 유효해서 굳이 대응할 필요는 없을 것 같습니다 ㅎㅎ<br />
일단 상정하기로는 static한 html string을 주로 사용할 목적이라서 event가 걸려있고, 동작이 많이 필요한 경우는 그냥 component 만드는걸 권장하는 방향으로 가이드 해야겠지요? ㅎㅎ

Reply:
static한 html string을 대응한다는 점에서 보면 기본 기능이 대체제로서 동작할 수 있는 느낌이 듭니다 🤔      
그래서, 간단하게 '이 컴퍼넌트를 언제 쓸지가 상상이 잘 안된다'라는 느낌이 있습니다

생각하시는 가장 뚜렷한 기능이 무엇인지 알 수 있을까요?

Reply:
1. vlossom 내부적으로 사용할 icon들을 svg html string으로 쓸건데, icon map을 만들고 아래 형태로 사용할 예정이구요.
```html
<vs-render :content="checkIcon" />
```

2. toast(snackbar) 컨텐츠에 component도 넣고 싶고, html string으로 (<span>hello <b>world!</b></span>) 넣고 싶을 때 사용할 수 있구요.

innerHtml 사용하지 않고 어떠한 형태로든 안전하게 render하고 싶을 때 쓸 목적으로 존재합니다